### ISSUE TITLE
setCenter called on child prop change?

### ISSUE THREAD

**Issue by GeoTob** [KoRiGaN/Vue2Leaflet#31]
I have a simple Button component, which extends `L.Control` and receives a `toggle` prop:

```
<v-map ...>
  <v-btn :toggle="mode" v-on:toggleEvent="toggleMode"></v-btn>
</v-map>
```
A click on the control fires `toggleEvent`, which toggles `mode` between `true` and `false` -- nothing more. 

I noticed that every time `mode` changes the map zooms and pans back into its initial position and zoom level. I checked the leaflet events without much luck, but just found out that I can avoid the problem by patching the `setCenter` method on my map with an empty function.

I'm not sure what exactly causes the call to `setCenter` in the first place and whether it's desired behaviour? Anyway to avoid this without removing the method?

**Comment by xemoka** [comment]
I'm just running into this problem now as well. I can't seem to find what's causing this either, although my case seems to be a bit different. 

Whenever a v-bind:style update is occurring on another element (sibling) the map is automatically returning to defaults. 

In this gist https://gist.github.com/xemoka/751a2f669685ccd1f17cc04a44c29f51 when line 6 is commented out, it doesn't occurr. 

I'm not sure why this would be happening, but it seems related...

**Comment by xemoka** [comment]
I ended up changing from using `:center` to `:bounds` on v-map and this was no longer a problem. 
I calculate bounds from my points that I add in `created()` (stored to this.bounds) and use `:bounds=bounds` when calling the map.

**Comment by KoRiGaN** [comment]
Seems related to #34.

@brianmay seems to have identified the problem and even give some workaround. It seems like @xemoka workaround. Create a array object instead of passing :center="[49.5, -117]".

**Comment by KoRiGaN** [comment]
This should be fixed in v0.0.49
